README.md

```diff
@@ -1,86 +1,90 @@
-# Kaigara
```

Comment:
I wonder why this whole file is changed again

Reply:
Added kaidump manual and modified secrets.yaml format